src/languages/en.js

```diff
@@ -135,6 +135,9 @@ export default {
     reportActionsView: {
         beFirstPersonToComment: 'Be the first person to comment',
     },
+    reportActionsViewMarkerBadge: {
+        newMsg: ({count}) => `${count} new messages`,
```

Comment:
Please make "messages" not plural if there's only 1 message

Reply:
what is the plural in spanish

Reply:
![Screen Shot 2021-08-13 at 4 36 36 PM](https://user-images.githubusercontent.com/3885503/129373683-d267aa36-9d2e-4e3e-b020-d1c32334e889.png)

Reply:
Looks like you already got it & updated the code - nice 👍